src/connectors/sort-by/connectSortBy.js

```diff
@@ -103,7 +103,8 @@ export default function connectSortBy(renderFn, unmountFn = noop) {
     return {
       $$type: 'ais.sortBy',
 
-      init({ helper, instantSearchInstance, parent }) {
+      init(initOptions) {
+        const { helper, instantSearchInstance, parent } = initOptions;
         const currentIndex = helper.state.index;
```

Comment:
We should get the current index from `getWidgetRenderState` otherwise we can get inconsistencies. We compute it twice at different spots here and we don't prevent the logic from differing.